src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const result = 40 * days;
```

Comment:
1. that's not result, I guess you can name it `rentalPrice` or smth
2. what's 40? I have no idea, just a number